website/client/src/components/tasks/task.vue

```diff
@@ -937,6 +940,27 @@ export default {
 
       return this.task.challenge.shortName ? this.task.challenge.shortName.toString() : '';
     },
+    isChallangeTask () {
+      return !isEmpty(this.task.challenge);
+    },
+    isGroupTask () {
+      return !isEmpty(this.task.group) && (this.task.group.taskId || this.task.group.id);
```

Comment:
`this.task.group` is always returned form the API even if empty so I think the first part of this check can be removed